xml_converter/src/attribute/specialization_filter.cpp

```diff
@@ -1,7 +1,7 @@
 #include "specialization_filter.hpp"
-
-SpecializationFilter parse_SpecializationFilter(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors) {
-    SpecializationFilter specialization_filter;
-    specialization_filter.parse(input, errors);
-    return specialization_filter;
-}
+//  Functionality will change for parsing
```

Comment:
No nee to comment this out, just delete it. If we need access to what it used to be we can look back into the git history.